readme.txt

```diff
@@ -71,6 +71,13 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 
 == Changelog ==
 
+= 0.25.1 2020-02-07 =

```

Comment:
Looks like this snuck in this branch on accident?

Reply:
When I released during the meetup I missed adding that to the 0.25.1 release https://github.com/woocommerce/woocommerce-admin/blob/version/1.0/readme.txt#L72 and it got included in this PR. Since this is a docs PR it should be okay to have it in this PR vs creating a separate one.

Reply:
@timmyc Can you confirm the above before I merge?

Reply:
Not ideal, but not a big deal either. I'm good with 🚢ing this.